osgp/platform/osgp-secret-management/src/integration-test/java/org/opensmartgridplatform/secretmanagement/application/SoapServiceSecretManagementIT.java

```diff
@@ -325,9 +326,12 @@ public void generateAndStoreSecretsAlreadyNewSecretPresent() throws IOException
     // Generate and store secret: this should NOT result in a fault message
     final Resource generateAndStoreRequest =
         new ClassPathResource("test-requests/generateAndStoreSecrets.xml");
-    this.mockWebServiceClient
-        .sendRequest(withSoapEnvelope(generateAndStoreRequest))
-        .andExpect(ResponseMatchers.noFault());
+    final ResponseActions response =
+        this.mockWebServiceClient
+            .sendRequest(withSoapEnvelope(generateAndStoreRequest))
+            .andExpect(ResponseMatchers.noFault());
+
+    assertThat(response).isNotNull();
```

Comment:
This check doesn't add anything because the response Matcher throws an assertion error instead of null.